src/utils/constants.js

```diff
@@ -14,19 +14,26 @@
  *
  */
 export const COMMAND_TYPES = [
-	'account',
-	'address',
-	'block',
-	'delegate',
-	'transaction',
+	'accounts',
+	'addresses',
+	'blocks',
+	'delegates',
+	'transactions',
 ];
 
-export const SINGULARS = {
-	accounts: 'account',
-	addresses: 'address',
-	blocks: 'block',
-	delegates: 'delegate',
-	transactions: 'transaction',
+export const PLURALS = {
```

Comment:
We could use some sanity check tests for this file.

Reply:
I will create the different issue for that!